pom.xml

```diff
@@ -187,49 +187,53 @@
     <testLogToFile>true</testLogToFile>
     <version.ant-contrib>1.0b3</version.ant-contrib>
     <version.asm>3.3.1</version.asm>
-    <version.com.github.spotbugs.spotbugs-maven-plugin>3.1.10</version.com.github.spotbugs.spotbugs-maven-plugin>
+    <version.com.github.spotbugs.spotbugs-maven-plugin>4.5.3.0</version.com.github.spotbugs.spotbugs-maven-plugin>
     <version.com.h2database>1.4.195</version.com.h2database>
     <version.com.ibm>11.5.0.0</version.com.ibm>
     <version.com.microsoft.sqlserver>mssql2005_sqljdbc_2.0</version.com.microsoft.sqlserver>
     <version.com.oracle>18.3.0.0</version.com.oracle>
+    <version.com.sun.corba.glassfish-corba-omgapi>3.2.0-b002</version.com.sun.corba.glassfish-corba-omgapi>
     <version.com.sun.grizzly>1.9.59</version.com.sun.grizzly>
     <version.com.sun.jersey>1.9.1</version.com.sun.jersey>
     <version.com.sybase.jConnect>6.0</version.com.sybase.jConnect>
     <version.commons-httpclient>3.1-jbossorg-1</version.commons-httpclient>
     <version.dom4j>1.6.1</version.dom4j>
     <version.doxia>1.8.1</version.doxia>
-    <version.fasterxml.jackson>2.12.3</version.fasterxml.jackson>
-    <version.glassfish.json>1.0.4</version.glassfish.json>
+    <version.fasterxml.jackson>2.13.1</version.fasterxml.jackson>
+    <version.glassfish.json>1.1.4</version.glassfish.json>
     <version.hamcrest>2.2</version.hamcrest>
     <version.io.mashona>1.0.0.Beta1</version.io.mashona>
     <version.io.narayana.checkstyle-config>1.0.1.Final</version.io.narayana.checkstyle-config>
     <version.jacorb>2.3.1jboss.patch01-brew</version.jacorb>
-    <version.jakarta.json-api>1.1.6</version.jakarta.json-api>
+    <version.jakarta.json-api>2.0.1</version.jakarta.json-api>
     <version.javax.annotation>1.2</version.javax.annotation>
     <version.javax.enterprise>2.0</version.javax.enterprise>
     <version.javax.inject>1</version.javax.inject>
-    <version.javax.javaee-api>7.0</version.javax.javaee-api>
+    <version.javax.javaee-api>8.0.1</version.javax.javaee-api>
     <version.javax.ws.rs-api>2.0.1.Final</version.javax.ws.rs-api>
+    <version.jaxws-ri>3.0.2</version.jaxws-ri>
```

Comment:
@mayankkunwar does JDK 17 require the major upgrade for JAX-WS?

Reply:
Yes @tomjenkinson upgrade is required, otherwise build was failing.

Reply:
Thanks @mayankkunwar I rebased on Friday and used a different branch for that given I don't have write access to where you raised your PR from. This has resulted in a second PR (https://github.com/jbosstm/narayana/pull/1981). I realise your PR (this one) was passing before but as it needed to rebasing and then after rebasing I started having some trouble with the JAX-WS version being 3.0.2 - it could be a problem with my rebasing.

What I am finding is that if I keep the version as it is in master (https://github.com/jbosstm/narayana/pull/1981/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8L213) then it does seem to work for both JDK 17 and 11 for the CORE profile so given it seems to work with 2.3.5 I am thinking we could just keep on that version?  If you are happy with 2.3.5 an option would be to close this PR and we can instead collaborate on the branch JBTM-3482 (you can find the commits here https://github.com/jbosstm/narayana/commits/JBTM-3482).

If you think 3.0.2 should be required may I ask you to rebase your branch too and see if you can get it to work? I think the need to manually rebase could perhaps be at least in part due to https://github.com/jbosstm/narayana/pull/1978/files.

Reply:
I see @tomjenkinson 
Earlier the version used for `jaxws-ri` was 2.3.0, and it was failing with that. If it works with 2.3.5 then I will just update it to the same.